### PR TITLE
MSVC/CMake Cruft: 39-backport of #4104

### PR DESCRIPTION
**Pull request by marcusmueller**

### CODE REVIEW

File: gr-blocks/lib/udp_sink_impl.cc [hunk 1]
```diff
@@ -101,13 +101,16 @@ int udp_sink_impl::work(int noutput_items,
                         gr_vector_void_star& output_items)
 {
     const char* in = (const char*)input_items[0];
-    ssize_t r = 0, bytes_sent = 0, bytes_to_send = 0;
-    ssize_t total_size = noutput_items * d_itemsize;
+    size_t r = 0;
+    std::ptrdiff_t bytes_sent = 0, bytes_to_send = 0;
+    size_t total_size = noutput_items * d_itemsize;
```
Comment (mbr0wn) [inline gr-blocks/lib/udp_sink_impl.cc:106]:
```suggestion
    const size_t total_size = noutput_items * d_itemsize;
```
...but I'm not a stickler for it.

File: gr-blocks/lib/ConfigChecks.cmake [hunk 2]
```diff
@@ -1,8 +1,8 @@
-# Copyright 2012 Free Software Foundation, Inc.
+#Copyright 2012 Free Software Foundation, Inc.
 #
-# This file is part of GNU Radio
+#This file is part of GNU Radio
 #
-# SPDX-License-Identifier: GPL-3.0-or-later
+#SPDX - License - Identifier : GPL - 3.0 - or -later
```
Comment (mbr0wn) [inline gr-blocks/lib/ConfigChecks.cmake:5]: Were these changes intentional?

Comment (marcusmueller) [reply gr-blocks/lib/ConfigChecks.cmake:5]: No, that was clang-format it seems which I let run on the CMake file to increase readability of the C++ code snippets

Comment (mbr0wn) [reply gr-blocks/lib/ConfigChecks.cmake]: I'm not a fan of these changes, but if we want them, maybe they should go onto their own commit. The comments look weird now, and I don't think the snippets required increased readability.

**Review by mbr0wn — APPROVED** [review]
I don't think I would have touched the cmake files with clang-format, and I would really prefer you revert the headers. Otherwise, this is a good change.

**Comment by marcusmueller** [comment]
Not quite sure what to do about this?

**Comment by marcusmueller** [comment]
Closing, let's first work out the master aspect of this.
